mod/groups/views/default/widgets/group_activity/edit.php

```diff
@@ -5,8 +5,12 @@
 
 $widget = elgg_extract('entity', $vars);
 
-// once autocomplete is working use that
-$groups = $widget->getOwnerEntity()->getGroups(['limit' => false]);
```

Comment:
I don't think so. Problem here is usage of getGroups() which only works for user entities. And getGroups() isn't used in the code of widgets.